api/envoy/extensions/outlier_detection_monitors/consecutive_errors/v3/consecutive_errors.proto

```diff
@@ -15,20 +15,18 @@ option java_multiple_files = true;
 option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/outlier_detection_monitors/consecutive_errors/v3;consecutive_errorsv3";
 option (udpa.annotations.file_status).package_version_status = ACTIVE;
 
+// [#protodoc-title: Consecutive errors monitor]
+// [#extension: envoy.outlier_detection_monitors.consecutive_errors]
+
 // Monitor which counts consecutive errors.
 // If number of consecutive errors exceeds the threshold, monitor will report that the host
 // is unhealthy.
-// [#not-implemented-hide:]
 message ConsecutiveErrors {
-  // Monitor name.
-  string name = 1;
+  // Base monitor config. Defines which results should be interpreted as errors
+  // and whether to enforce ejection when number of errors exceeds defined threshold.
+  common.v3.MonitorCapture errors = 1 [(validate.rules).message = {required: true}];
 
   // The number of consecutive errors before ejection occurs.
+  // Defaults to 3.
   google.protobuf.UInt32Value threshold = 2 [(validate.rules).uint32 = {lte: 100}];
-
-  // The % chance that a host is actually ejected. Defaults to 100.
-  google.protobuf.UInt32Value enforcing = 3 [(validate.rules).uint32 = {lte: 100}];
-
-  // Error buckets.
-  common.v3.ErrorBuckets error_buckets = 4;
```

Comment:
Given this extension as specific example, only threshold and enforcing is necessary. ErrorBuckets/errors are unecessary now.